c2cgeoportal/tests/functional/test_themes.py

```diff
@@ -282,11 +290,11 @@ def test_min_levels(self):
         themes = entry.themes()
         self.assertEquals(self._get_filtered_errors(themes), set([
             u"The Layer '__test_theme/__test_layer_group_1/__test_layer_internal_wms' is under indented (1/2).",
-            u"Layer '__test_layer_external_wms' cannot be in the group '__test_layer_group_1' (internal/external mix).",
-            u"Layer '__test_layer_wmts' cannot be in the group '__test_layer_group_1' (internal/external mix).",
-            u"Layer '__test_layer_wmts' cannot be in the group '__test_layer_group_2' (internal/external mix).",
+            u"The Layer '__test_theme/__test_layer_group_1/__test_layer_wmts' is under indented (1/2).",
+            u"The Layer '__test_theme/__test_layer_group_2/__test_layer_external_wms' is under indented (1/2).",
```

Comment:
Those messages looks strange it seem that the indent incrementation don't works anymore!